src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VedtakRepository.kt

```diff
@@ -283,7 +283,7 @@ class VedtakRepository(private val database: DatabaseInterface) : IVedtakReposit
 
         private const val GET_UNPUBLISHED_INFOTRYGD =
             """
-                SELECT * FROM VEDTAK WHERE published_infotrygd_at IS NULL
+                SELECT * FROM VEDTAK WHERE published_infotrygd_at IS NULL AND created_at < now() - interval '10 minutes'
```

Comment:
Diskuterte dette litt med @geir-waagboe. Siden vi fortsatt har en cronjob kjørende hvert minutt som plukker opp upubliserte vedtak var tanken å sjekke `created_at` her for å unngå at api-kallet fra frontend som skal sende vedtak til infotrygd går i beina på cronjobben.